client/src/components/TourInfoDialog/TourInfoDialog.js

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable react/forbid-prop-types */
```

Comment:
this is already included in the eslint config file as a general rule.   you can remove this line

Reply:
i don’t  see it in eslint config